boards/qemu_cortex_a53.overlay

```diff
@@ -49,6 +49,14 @@
 			};
 		};
 	};
+
```

Comment:
I'm not sure we need qemu configuration. There is a problem that it is very hard to start OP-TEE with zephyr on QEMU until we will have UEFI support in Zephyr. So I suggest removing this config.

Reply:
Ok. Deleted